blocks/footer/footer.js

```diff
@@ -51,10 +51,14 @@ function decorateFooter(block) {
  */
 export default async function decorate(block) {
   block.textContent = '';
+  let footerPath = 0;
 
   // fetch footer content
   const footerMeta = getMetadata('footer');
-  const footerPath = footerMeta || (getLanguage() === 'jp' ? '/footer' : `/${getLanguage()}/footer`);
+  const customNavFooter = getMetadata('customnavfooter');
```

Comment:
If this is for dentistry header/footers, rather than being driven by metadata, we can based it on the path if the path always contains `/dentistry` . Reduces the overhead a bit for authors for taking care of specifying header/footer.